File: DSA/VikrantSinghBanwala_590018082/day32_question3.c
#include<stdio.h>
int findXOR(int arr[], int size) {
    int result = 0;
    for (int i = 0; i < size; i++) {
        result ^= arr[i];
    }
    return result;
}
int main(){
    int arr1[] = {1, 2, 3};
    printf("%d\n", findXOR(arr1, 3));
    int arr2[] = {4, 5, 6};
    printf("%d\n", findXOR(arr2, 3));
    return 0;
}

File: DSA/NiharikaChand_590017697/Day32Question3.java
import java.util.*;
class Day32Question3
{
    public static int findXOR(int arr[]) 
    {
        int result = 0;
        for (int num : arr) 
        {
            result ^= num;
        }

        return result;
    }

    public static void main(String args[]) 
    {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter size of array: ");
        int n = sc.nextInt();
        int arr[] = new int[n];
        System.out.print("Enter array elements: ");
        for (int i = 0; i < n; i++) 
        {
            arr[i] = sc.nextInt();
        }

        int result = findXOR(arr);
        System.out.println("Output: " + result);
    }
}
